triplets/arm64-osx.cmake

```diff
@@ -0,0 +1,6 @@
+set(VCPKG_TARGET_ARCHITECTURE arm64)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE dynamic)
```

Comment:
It's probably a good idea to call the triplet `arm64-osx-dynamic` to distinguish it from the built-in arm64-osx triplet. As is, it replaces a built-in, which is probably not a great idea, and doesn't describe what it's customised.